src/components/pages/callouts/CalloutIntroPanel.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <CalloutSidePanel :show="true" @close="$emit('close')">
```

Comment:
Should this not be being bound to a prop called `show`? Does the panel ever close at the mo?

Reply:
Or maybe this is working because Vue is automatically passing the `show` prop from the parent? In which case I think it would be less confusing to pass the `show` value explicitly.

Reply:
Makes total sense. I added the prop `show` to CalloutIntroPanel, and changed the element property to `:show="show"` -- just making sure this is the right approach?